cmd/kubeadm/.import-restrictions

```diff
@@ -0,0 +1,260 @@
+{
+	"Rules": [{
+		"SelectorRegexp": "k8s[.]io",
+		"AllowedPrefixes": [
+			"k8s.io/api/admission/v1beta1",
```

Comment:
Can't the prefixes be pruned down to the lowest common denominator for a bunch of the client stuff.  

e.g. k8s.io/api, k8s.io/apimachinery ...

Reply:
/approve

Reply:
yep, will tweak over the weekend.